doc/guides/writing-tests.md

```diff
@@ -18,6 +18,13 @@ Add tests when:
 - Fixing regressions and bugs.
 - Expanding test coverage.
 
+## Where does the test go
```

Comment:
This would sound better as "Where are tests located" or even "Test directory structure".

Reply:
changing it to "Test directory structure"